src/panels/lovelace/hui-root.ts

```diff
@@ -405,11 +405,25 @@ class HUIRoot extends LitElement {
 
     const viewPath = this.route!.path.split("/")[1];
 
+    const isVisible = (viewIndex) =>
```

Comment:
```suggestion
    const isVisible = (view) =>
```

Reply:
After this suggested change the patch doesn't work. Without it works fine.

Reply:
You would need to rename the var name in the function too